app/src/main/java/com/arctouch/movies/movie/business/AppMovieBO.java

```diff
@@ -0,0 +1,66 @@
+package com.arctouch.movies.movie.business;
+
+import android.support.annotation.NonNull;
+
+import com.arctouch.movies.movie.dao.AppMovieDAO;
+import com.arctouch.movies.movie.dao.MovieDAO;
+import com.arctouch.movies.movie.model.Movie;
+import com.arctouch.movies.movie.model.MovieDetails;
+import com.arctouch.movies.serverapi.response.MovieResponse;
+
+import java.util.List;
+
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+
+/**
+ * Created by marcelo.alves on 06/09/17.
+ */
+
+public class AppMovieBO implements MovieBO {
```

Comment:
This class is nothing but a wrapper over movieDAO, isn't it? Why not call AppMovieDAO directly?

Reply:
I like to keep the structure of BO and DAO, thinking in the future where we can have more logic into BO